web/src/components/FeedbackButtons.tsx

```diff
@@ -19,23 +20,32 @@ type FeedbackButtonsProps = {
 
 const FeedbackButtons = ({ isPositive, onRatingChange }: FeedbackButtonsProps): ReactElement => {
   const { t } = useTranslation('feedback')
+
+  const handleChange = (event: React.MouseEvent<HTMLElement>, newValue: string | null) => {
+    if (newValue === t('useful')) {
```

Comment:
🔧 I think it would be better to set an explicit value instead of falling back to the text property.